src/PDFsharper.UnitTests/Drawing/FontHelperTests.cs

```diff
@@ -0,0 +1,313 @@
+            Assert.IsTrue(size.Height > 0, "size.Height should be greater than zero");
+        }
+
+        [TestMethod]
+        public void MeasureString_TimesBold()
+        {
+            XFont font = new XFont("Times-Roman", 10.0, XFontStyle.Bold);
+            XSize size = FontHelper.MeasureString("test", font);
+
+            Assert.IsNotNull(size, "size should not be null");
+            Assert.IsTrue(size.Width > 0, "size.Width should be greater than zero");
+            Assert.IsTrue(size.Height > 0, "size.Height should be greater than zero");
+        }
+
+        [TestMethod]
+        public void MeasureString_TimesBoldItalic()
+        {
+            XFont font = new XFont("Times-Roman", 10.0, XFontStyle.BoldItalic);
+            XSize size = FontHelper.MeasureString("test", font);
+
+            Assert.IsNotNull(size, "size should not be null");
+            Assert.IsTrue(size.Width > 0, "size.Width should be greater than zero");
+            Assert.IsTrue(size.Height > 0, "size.Height should be greater than zero");
+        }
+
+        [TestMethod]
+        public void MeasureString_TimesItalic()
+        {
+            XFont font = new XFont("Times-Roman", 10.0, XFontStyle.Italic);
+            XSize size = FontHelper.MeasureString("test", font);
+
+            Assert.IsNotNull(size, "size should not be null");
+            Assert.IsTrue(size.Width > 0, "size.Width should be greater than zero");
+            Assert.IsTrue(size.Height > 0, "size.Height should be greater than zero");
+        }
+
+        [TestMethod]
+        public void MeasureString_TimesNewRoman()
+        {
+            XFont font = new XFont("Times New Roman", 10.0);
+            XSize size = FontHelper.MeasureString("test", font);
+
+            Assert.IsNotNull(size, "size should not be null");
+            Assert.IsTrue(size.Width > 0, "size.Width should be greater than zero");
+            Assert.IsTrue(size.Height > 0, "size.Height should be greater than zero");
+        }
+
+        [TestMethod]
+        public void MeasureString_TimesRoman()
+        {
+            XFont font = new XFont("Times-Roman", 10.0);
+            XSize size = FontHelper.MeasureString("test", font);
+
+            Assert.IsNotNull(size, "size should not be null");
+            Assert.IsTrue(size.Width > 0, "size.Width should be greater than zero");
+            Assert.IsTrue(size.Height > 0, "size.Height should be greater than zero");
+        }
+
+        [TestMethod]
+        public void MeasureString_ZapfDingbats()
+        {
+            XFont font = new XFont("ZapfDingbats", 10.0);
+            XSize size = FontHelper.MeasureString("test", font);
+
+            Assert.IsNotNull(size, "size should not be null");
+            Assert.IsTrue(size.Width > 0, "size.Width should be greater than zero");
+            Assert.IsTrue(size.Height > 0, "size.Height should be greater than zero");
+        }
+
+        [TestMethod]
+        public void MeasureString()
+        {
```

Comment:
This should differ from the split out tests or be removed.  Should the tests look for a specific size vs > 0  ?